### ISSUE TITLE
[Supporting the platform- Pick a sample as a starting point window]: At 400% zoom, All content present on "Pick a sample as a starting point" window are not visible.

### ISSUE THREAD

**Issue by jwoo-msft** [microsoft/AdaptiveCards#6954]
https://microsoft.visualstudio.com/OS/_workitems/edit/30645314
